### PR TITLE
Fixing NaN issue in summary histogram

### PR DESCRIPTION
**Pull request by cpheinrich**
- Adjusting small offset parameter in denominator of loss function
- Commenting out the summary since we are not currently reading these files anyway

### CODE REVIEW

File: mvsnet/loss.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def non_zero_mean_absolute_diff(y_true, y_pred, interval):
         shape = tf.shape(y_pred)
         interval = tf.reshape(interval, [shape[0]])
         mask_true = tf.cast(tf.not_equal(y_true, 0.0), dtype='float32')
-        denom = tf.reduce_sum(mask_true, axis=[1, 2, 3]) + 1e-7
+        denom = tf.abs(tf.reduce_sum(mask_true, axis=[1, 2, 3])) + 1e-6
```
Comment (cpheinrich) [inline mvsnet/loss.py:20]: Taking the absolute value, and increasing epsilon, should prevent division by near-zero numbers

**Review by indianajohn — APPROVED** [review]
I won't pretend to have a lot of context here, but the changes make sense in terms of your comments.
